docs/resources/compute_keypair_v2.md

```diff
@@ -38,6 +41,12 @@ The following attributes are exported:
 
 * `public_key` - See Argument Reference above.
 
+* `private_key` - The information about the private for an SSH key.
+
+->
+The information about the `private_key` is contained in the response for **creating** an SSH key.
+The information about the `private_key` is not contained in the response for **importing** an SSH key.
```

Comment:
It's better to also say that the private key is returned only if the public key was not set originally

Reply:
Also, `is contained` sounds clumsy :slightly_smiling_face: Better to put the phrase in the active voice.